lib/puppet/face/config.rb

```diff
@@ -129,4 +129,45 @@
       nil
     end
   end
+
+  action(:delete) do
+    summary _("Delete a Puppet setting.")
+    #TRANSLATORS '--section' is a command line option and should not be translated
```

Comment:
Is this comment associated with the notes string or example string below, instead?